iaso/api/microplanning.py

```diff
@@ -164,6 +164,15 @@ def validate(self, attrs):
         return validated_data
 
 
+class TeamManagersFilterBackend(filters.BaseFilterBackend):
+    def filter_queryset(self, request, queryset, view):
+        managers = request.GET.get("managers", None)
+        if managers:
+            manager_ids = [int(val) for val in managers.split(",") if val.isnumeric()]
+            return queryset.filter(manager_id__in=manager_ids)
+        return queryset
```

Comment:
Just a quick comment but nothing blocking: I'm in favor of using the "default" Django format instead of sending params "comma separated":

```
GET /api/endoint/?tag=tag1&tag=tag2
# VS
GET /api/endoint/?tag=tag1,tag2
```

With this format, we can often have a simpler filter and the validation for free!

See the [Slack discussion](https://bluesquare.slack.com/archives/C032F8GFAUD/p1716904787344399).

Reply:
Thanks, I didn't know about this discussion :+1: 

Honestly, I don't mind either way (I used as a reference the multiple users in the potential payments page). However, I think we should definitely have a single way of managing multiple selects throughout the system.